app-config.yaml

```diff
@@ -373,6 +373,11 @@ auth:
       development: {}
 costInsights:
   engineerCost: 200000
+  baseCurrency:
+    locale: en-US
+    options:
+      currency: EUR
```

Comment:
should be USD :)

Reply:
oh, sorry, forgot to remove after testing it 🙈